src/colorizer/utils/dataset_utils.ts

```diff
@@ -106,7 +107,8 @@ export const updateManifestVersion = (manifest: AnyManifestFile): ManifestFile =
       features.push({
         name: featureName,
         data: featurePath,
-        units: featureMetadata?.units || undefined,
+        // Note change from "units" to "unit"
+        unit: featureMetadata?.units || undefined,
```

Comment:
Remaps from old metadata format to newer.

Reply:
Any reason to check for `featureMetadata?.unit` here as well, or will we definitely only see `units` if we've made it here?